### DISCUSSION TITLE
If I want to achieve second-level metric monitoring, what code modifications should I make

### DISCUSSION THREAD

**Discussion by hxd123456** [apache/skywalking#10701] [Q&A] [answered]
If I want to achieve second-level metric monitoring, what code modifications should I make
I want to store more finely divided indexes and place unfinished metrics in Java's off heap memory to achieve second-level  metric monitoring
but I'm not sure if this will achieve my goal

**Comment by wu-sheng** [comment, marked as answer]
This is not SkyWalking scope. We don't have this concept.
